internal/commands/pull/command.go

```diff
@@ -138,6 +165,14 @@ func (cmd *Command) Handler(profile *user.Profile, ui terminal.UI, clients cli.C
 		ui.Print(terminal.NewDebugLog("Fetched hosting assets"))
 	}
 
+	for templateID, templateZipPkg := range clientZipPkgs {
```

Comment:
if we wanted, right before this would be where I'd add `checkPathDestination` for the `pathFrontend` filepath